src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -161,70 +187,78 @@ const update = (participantProjectSpecies: ParticipantProjectSpecies): Promise<R
 };
 
 const getProjectsForSpecies = async (speciesId: number, organizationId: number): Promise<SpeciesProjectsResult[]> => {
+  const todayUTC = DateTime.fromISO(new Date().toISOString()).toFormat('yyyy-MM-dd');
+  console.log('todayUTC', todayUTC);
   const searchParams: SearchRequestPayload = {
-    prefix: 'species',
+    prefix: 'projects.projectDeliverables',
     fields: [
-      'participantProjectSpecies.id',
-      'participantProjectSpecies.project.name',
-      'participantProjectSpecies.project.id',
-      'participantProjectSpecies.submissionStatus',
+      'id',
+      'type',
+      'project.participantProjectSpecies.id',
+      'project.participantProjectSpecies.project.name',
+      'project.participantProjectSpecies.project.id',
+      'project.participantProjectSpecies.species.id',
+      'project.participantProjectSpecies.species.organization.id',
+      'project.participantProjectSpecies.submissionStatus',
+      'module.cohortModules.startDate',
+      'module.cohortModules.endDate',
     ],
     search: {
       operation: 'and',
       children: [
         {
           operation: 'field',
-          field: 'organization_id',
+          field: 'type',
+          type: 'Exact',
+          // Since the search API is localized, we need to send the localized values
+          // ** This will not work as expected if the BE and FE localized values do not match
+          values: [getDeliverableTypeLabel('Species' as DeliverableTypeType)],
```

Comment:
That's unfortunate this is somewhat brittle, but that's how it goes sometimes 🤷

Reply:
Yea we've talked about having a unified localization dictionary but the juice doesn't seem to be worth the squeeze at the moment. I suspect this will change when we add several more languages.